solr/solrj/src/java/org/apache/solr/client/solrj/SolrRequest.java

```diff
@@ -291,6 +292,15 @@ public final T process(SolrClient client, String collection)
     return typedResponse;
   }
 
+  /**
+   * @lucene.experimental
+   */
+  public final T processWithBaseUrl(HttpSolrClientBase client, String url, String collection)
```

Comment:
Added this in kind with other process method that returns `T`.  This enables most users of the signature change I did to switch to this instead.  I'm pretty happy with this.